tsconfig.json

```diff
@@ -1,15 +1,25 @@
 {
     "compilerOptions": {
-        "esModuleInterop": true,
+        // "esModuleInterop": true,
```

Comment:
Why?

Reply:
I must admit that my understanding of this option is not good. While adding Wasm compatibility and switching over to Webpack, I found it necessary to change the output format of the bundler/transpiler to be Node16, which I think is more modern than CommonJS. My hand-wavy answer is that this probably made esModuleInterop unnecessary by converting everything to a common modules format.